editor/editor_node.cpp

```diff
@@ -3617,7 +3617,8 @@ void EditorNode::set_current_scene(int p_idx) {
 	_update_title();
 	_update_scene_tabs();
 
-	call_deferred(SNAME("_set_main_scene_state"), state, get_edited_scene()); // Do after everything else is done setting up.
+	if (tabs_to_close.is_empty())
+		call_deferred(SNAME("_set_main_scene_state"), state, get_edited_scene()); // Do after everything else is done setting up.
```

Comment:
```suggestion
	if (tabs_to_close.is_empty()) {
		call_deferred(SNAME("_set_main_scene_state"), state, get_edited_scene()); // Do after everything else is done setting up.
	}
```

This doesn't follow our codestyle. Curly brackets are mandatory.